lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -3121,22 +3167,41 @@ LogicalResult StmtEmitter::visitSV(IfOp op) {
 
   indent() << "if (";
 
-  // If we have an else and and empty then block, emit an inverted condition.
-  if (!op.hasElse() || !op.getThenBlock()->empty()) {
+  // In the loop, emit an if statement assuming the keyword introducing
+  // it (either "if (" or "else if (") was printed already.
+  IfOp ifOp = op;
+  for (;;) {
+    // If we have an else and and empty then block, emit an inverted condition.
```

Comment:
This should happen in prepare or, even better, in SV canonicalization.

Reply:
I'll address this in a different PR.